### PR TITLE
Catch exception when trying to show TPR unit image

### PR DESCRIPTION
**Pull request by tuutti**
## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-X-catch-file-exception`
* Run `make drush-cr`

### CODE REVIEW

**Review by Arkkimaagi — APPROVED** [review]
This fixes the problem on my machine.

File: modules/hdbt_content/hdbt_content.tokens.inc [hunk 1]
```diff
@@ -60,7 +60,14 @@ function hdbt_content_tokens(
     if ($name === 'picture' && $type === 'tpr_unit' && !empty($data[$type])) {
       /** @var \Drupal\helfi_tpr\Entity\Unit $entity */
       $entity = $data[$type];
-      $replacements[$original] = $entity->getPictureUrl() ?? $default_image;
+      $image = NULL;
+
+      try {
+        $image = $entity->getPictureUrl();
+      }
+      catch (\Exception) {
```
Comment (ltpk) [inline modules/hdbt_content/hdbt_content.tokens.inc:68]: Is it necessary to catch an exception here? Could we just check whether `$entity->getPictureUrl()` returns `FALSE`?

Comment (tuutti) [reply modules/hdbt_content/hdbt_content.tokens.inc:68]: No, but this exception check could be moved to TPR module instead: https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/blob/main/src/Entity/Unit.php#L167

**Review by ltpk — APPROVED** [review]

**Comment by khalima** [comment]
Closing as the actual fix is done in https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/121

**Pull request closed without merging**